src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = '' } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId) || null;
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                data-cy="Tab"
+                className={selectedTab?.id === tab.id ? 'is-active' : ''}
+              >
+                <Link to={`/tabs/${tab.id}`}>{tab.title}</Link>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+
+      {!selectedTab
+        && (
+          <div className="block" data-cy="TabContent">
+            Please select a tab
+          </div>
+        )}
+
+      <div className="block" data-cy="TabContent">
+        {selectedTab?.content}
+      </div>
```

Comment:
This block you can render only if `selectedTab` exists (as you do above for `!seltctedTab`)